app.py

```diff
@@ -0,0 +1,42 @@
+import psycopg2
+import os
+from flask import Flask, jsonify
+from cfenv import AppEnv
+from psycopg2.extras import RealDictCursor
+from flask_cors import CORS
+
+
+app = Flask(__name__)
+app_env = AppEnv()
+CORS(app, origins=['https://federalist-31c21015-f923-4733-95df-2e979da3e393.sites.pages.cloud.gov'], headers=['Content-Type'], methods=['GET'])
```

Comment:
Instead of hard coding the origin, lets use an environment variable we can pass to the deployment.

Reply:
are there alternatives to `exporting` the env var from a bash locally?

Reply:
When you push an app you can add an environment variable so the app can use it. 

ie
```
cf push my-app --var CORS_ORIGIN=my.origin.gov
```

And then in your app, you can read it from the os env.
```python
origin = os.getenv('CORS_ORIGIN')
```